ocs_ci/deployment/deployment.py

```diff
@@ -418,10 +421,21 @@ def deploy_ocs_via_operator(self):
             run_cmd(f"oc create -f {self.CUSTOM_STORAGE_CLASS_PATH}")
 
         # creating StorageCluster
-        if self.platform == constants.IBM_POWER_PLATFORM:
-            cluster_data = templating.load_yaml(constants.IBM_STORAGE_CLUSTER_YAML)
+        if config.ENV_DATA.get("enable_flexible_scaling"):
+            if ocs_version < 4.7:
+                error_message = "Flexible scaling can be enabled only on OCS >= 4.7!"
+                logger.error(error_message)
+                raise UnsupportedFeatureError(error_message)
+            else:
+                logger.info("Enabling flexible scaling!")
+                cluster_data = templating.load_yaml(
+                    constants.FLEXIBLE_STORAGE_CLUSTER_YAML
+                )
```

Comment:
is  constants.FLEXIBLE_STORAGE_CLUSTER_YAML works for IBM_POWER_PLATFORM ?

Reply:
I didn't understand this.

Reply:
since we are skipping https://github.com/red-hat-storage/ocs-ci/pull/3485/files#diff-8571fc52f1040ac2cb01985da6d61cc93a535f812814c200b45d00daf28c2e3fR435 for flexible_scaling, storageCluster CR is different for IBM platform.

Usecase: If  flexible_scaling is enable for IBM platform, we are skipping https://github.com/red-hat-storage/ocs-ci/pull/3485/files#diff-8571fc52f1040ac2cb01985da6d61cc93a535f812814c200b45d00daf28c2e3fR435 and loads the normal STORAGE_CLUSTER_YAML

Reply:
flexible scaling is targeted to bare metal installations and IBM is one of the core use cases. I would like some clarification on what needs to be changed in that case.

Reply:
@gitsridhar  Could you check https://github.com/red-hat-storage/ocs-ci/pull/3485/files#diff-8a5c1c74a1286c016ee290ca3ef9085364965a6edc0fa6a0fb91541960f3dfbd works for IBM Power platform for flexible scaling